### ISSUE TITLE
Route Editing Tab could not be selected And Bridge status is Null

### ISSUE THREAD

**Issue by kanbei** [lgsvl/simulator#550]
Hello. I launched  Apollo and the Simulator according to the direction on website,
but in  Apollo, the Route Editing Tab could not be selected. 
In the Simulator, bridge status is Null.
I have input `bash scripts/bridge.sh`, like this photo, but I `m not sure whether the bridge is  launched successfully.
So how could I solve the problem.
![Screenshot from 2019-12-09 18-59-14](https://user-images.githubusercontent.com/14128881/70431362-3eed3f00-1ac0-11ea-9c77-13c5d5bad0fb.png)
![Screenshot from 2019-12-06 20-05-53](https://user-images.githubusercontent.com/14128881/70431350-36950400-1ac0-11ea-8fc1-8ccd7d8196d7.png)

**Comment by OutOfCage** [comment]
Did you add the sensor configuration for your vehicle? 
[How to add vehicle](https://www.lgsvlsimulator.com/docs/vehicles-tab/)
And did you set the host and port of the bridge when adding a simulation?
[How to add simulation](https://www.lgsvlsimulator.com/docs/simulations-tab/#how-to-addedit-a-simulation)

**Comment by kanbei** [comment]
Thank you. I have followed the instruction, and set senor configuration and port of the bridge, Apollo and the simulation was connected well. But now still have 3 problems.
The first one is if I set JSON Configuration of camera, according to the example(https://www.lgsvlsimulator.com/docs/apollo5-0-json-example/), the simulation was crashed suddenly after I created the simulation and clicked 'play' button. 
When I deleted the configuration of camera, the simulation worked well. How could  I set the configuration of camera?

The second one is the perception and traffic light module can't be enabled in Apollo. I tried to click them but soon after it became unable. 

The third problem is when I added point and send routing request, the car didn't run and was still in the initial position. 
I think the position information in the simulation can be send to Apollo successfully because in Apollo,  the car could run well while I was operating the car manually in the simulation. But I don't know why car can't run automatically.
![Screenshot from 2019-12-17 19-34-49](https://user-images.githubusercontent.com/14128881/70992997-58b90280-210e-11ea-80d9-6487911202f0.png)

**Comment by kanbei** [comment]
> Did you add the sensor configuration for your vehicle?
> How to add vehicle
> And did you set the host and port of the bridge when adding a simulation?
> How to add simulation

Thank you. I have followed the instruction, and set senor configuration and port of the bridge, Apollo and the simulation was connected well. But now still have 3 problems.

The first one is if I set JSON Configuration of camera, according to the example(https://www.lgsvlsimulator.com/docs/apollo5-0-json-example/), the simulation was crashed suddenly after I created the simulation and clicked 'play' button.
When I deleted the configuration of camera, the simulation worked well. How could I set the configuration of camera?

The second one is the perception and traffic light module can't be enabled in Apollo. I tried to click them but soon after it became unable.

The third problem is when I added point and send routing request, the car didn't run and was still in the initial position.
I think the position information in the simulation can be send to Apollo successfully because in Apollo, the car could run well while I was operating the car manually in the simulation. But I don't know why car can't run automatically.
![70992997-58b90280-210e-11ea-80d9-6487911202f0](https://user-images.githubusercontent.com/14128881/71014991-30dd9500-2136-11ea-9662-8a4eea902ad3.png)
